cheatsheets/Multifactor_Authentication_Cheat_Sheet.md

```diff
@@ -255,7 +255,7 @@ Smartcards are credit-card size cards with a chip containing a digital certifica
 
 ### SMS Messages and Phone Calls
 
-SMS messages or phone calls can be used to provide users with a single-use code that they must submit as an additional factor. Due to the cons posed by these methods, they should not be used to protect applications that hold Personally Identifiable Information (PII) or where there is financial risk. e.g. Healthcare and Banking. [NIST SP 800-63](https://pages.nist.gov/800-63-3/sp800-63b.html) does not allow these factors for applications containing PII.
+SMS messages or phone calls can be used to provide users with a single-use code that they must submit as an additional factor. Due to the risks posed by these methods, they should not be used to protect applications that hold Personally Identifiable Information (PII) or where there is financial risk. e.g. healthcare and banking. [NIST SP 800-63](https://pages.nist.gov/800-63-3/sp800-63b.html) does not allow these factors for applications containing PII.
```

Comment:
Since we are making this change, can we broaden banking to maybe stating something like "the financial exchange market" or something similar?

Reply:
Personally I think "where there is financial risk" is enough

Reply:
@szh wrote:
> Personally I think "where there is financial risk" is enough

It's ambiguous though, unless we change it to "where there is financial risk to customers / victims". There is almost always some financial risk to the company whose application gets breached though, even from reputational impacts (which can result in loss of company stock prices, loss of revenue because customers loss confidence, etc.) If we want to quantify it, sure. OTOH, as I said, it's not a showstopper for me.